src/scripts/clipperUI/componentBase.ts

```diff
@@ -6,7 +6,15 @@ export interface EnableInvokeParams {
 	tabIndex?: number;
 	args?: any;
 	idOverride?: string;
-	setNameForArrowKeyNav?: string;
+}
+
+export interface EnableAria {
```

Comment:
There are repeated common properties between this new interface and the EnableInvokeParams